lang/java/reef-applications/reef-vortex/pom.xml

```diff
@@ -51,48 +51,20 @@ under the License.
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.apache.avro</groupId>
-            <artifactId>avro</artifactId>
+            <groupId>com.esotericsoftware</groupId>
+            <artifactId>kryo-shaded</artifactId>
```

Comment:
I filled [REEF-1337](https://issues.apache.org/jira/browse/REEF-1337) to track that. It shouldn't block this PR.